RELEASE/scripts/sl_ascend.ash

```diff
@@ -12703,8 +12704,57 @@ boolean L9_chasmBuild()
 		return true;
 	}
 
+
+	// -Combat is useless here since NC is triggered by killing Orcs...So we kill orcs better!
+	asdonBuff($effect[Driving Intimidatingly]);
+
+	// Check our Load out to see if spells are the best option for Orc-Thumping
+	if(setFlavour($element[cold]) && canUse($skill[Stuffed Mortar Shell]))
+	{
+		set_property("sl_useSpellsInOrcCamp", true);
+	}
+
+	if(setFlavour($element[cold]) && canUse($skill[Cannelloni Cannon]))
+	{
+	set_property("sl_useSpellsInOrcCamp", true);
+	}
+
+	if(canUse($skill[Saucegeyser]))
+	{
+		set_property("sl_useSpellsInOrcCamp", true);
+	}
+
+	if(canUse($skill[Saucecicle]))
+	{
+		set_property("sl_useSpellsInOrcCamp", true);
+	}
+
+	// Always Maximize and choose our default Non-Com First, in case we are wrong about the non-com we MAY have some gear still equipped to help us.
+	if(get_property("sl_useSpellsInOrcCamp").to_boolean())
+	{
+		print("Preparing to Blast Orcs with Cold Spells!", "blue");
+		addToMaximize("myst,20spell damage,40spell damage percent,20cold spell damage,-100 ml");
+		buffMaintain($effect[Carol of the Hells], 50, 1, 1);
+		buffMaintain($effect[Song of Sauce], 150, 1, 1);
+
+		// Since we are Buffing for Spells and Myst, set choice adventure just in case
+		set_property("choiceAdventure1345", 2);
```

Comment:
I would rather the choice adventure be set to 0 for now, so that amisprediction won't be squashed.